### PR TITLE
changelogs

### PR DESCRIPTION
**Pull request by felixgehrmann**
### All changes meet the following requirements
- [ ] Changelog entry was added
- [ ] Changes have been documented
- [ ] Changes have been tested by the author
- [ ] Changes have been tested by the reviewer
- [ ] Plugin can be built

@plentymarkets/plentyshop

### CODE REVIEW

File: meta/documents/changelog_de.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+
+### Geändert
+
+- Das Anklicken der Artikelbilder im Bewertungswidget für die Bestellbestätigung leitet nun auf die Artikelseite weiter.
+  Wir bedanken uns bei User @MaxBentz für den Beitrag.
+
+### Behoben
+
+- Beim Editieren von einer Bewertung war die Reihenfolge der Bewertungssterne falsch. Dies wurde behoben.
+  Wir bedanken uns bei User @MaxBentz für den Beitrag.
```
Comment (Dominik2809) [inline meta/documents/changelog_de.md:13]: Enter needed

**Review by Dominik2809 — CHANGES_REQUESTED** [review]

File: meta/documents/changelog_de.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
 # Release Notes für Feedback
 
+## v4.0.7 (2022-10-21)
```
Comment (Dominik2809) [inline meta/documents/changelog_de.md:3]: Git compare link needed in public plugins

**Review by Dominik2809 — CHANGES_REQUESTED** [review]

File: meta/documents/changelog_de.md (hunk 1 above)
Comment (felixgehrmann) [reply meta/documents/changelog_de.md:13]: done

File: meta/documents/changelog_de.md (hunk 2 above)
Comment (felixgehrmann) [reply meta/documents/changelog_de.md]: done

**Review by Dominik2809 — APPROVED** [review]

**Pull request merged**